libr/util/annotated_code.c

```diff
@@ -23,6 +23,14 @@ R_API void r_annotation_free(void *e, void *user) {
 	}
 }
 
+R_API bool is_annotation_reference(RCodeAnnotation *annotation) {
```

Comment:
Please consider renaming these functions. Most R_API functions use common prefix, this helps avoiding naming conflict and improves discoverability. Maybe `r_annotation_is_reference` and `r_annotation_is_variable`.

Reply:
I will rename them.

Reply:
@karliss I have renamed functions. If you are fine with the code now, please merge it. I think [PR #2352](https://github.com/radareorg/cutter/pull/2352) Cutter can get merged tonight. Since it uses the new API implemented in this PR, I want this to get merged first.

I will send a separate PR with documentation using Doxygen tonight, but I think it can be another PR as it could take more time to review.